falcon/request.py

```diff
@@ -683,6 +687,22 @@ def access_route(self):
     def remote_addr(self):
         return self.env.get('REMOTE_ADDR')
 
+    @property
+    def port(self):
+        try:
+            host_header = self.env['HTTP_HOST']
+            host, port = parse_host(host_header)
```

Comment:
If port isn't present, we should probably return the default port based on the schema, in oder to normalize what would be returned with the `SERVER_PORT` branch. What do you think?